feature-libs/product-configurator/rulebased/components/form/configurator-form.component.ts

```diff
@@ -96,4 +96,10 @@ export class ConfiguratorFormComponent implements OnInit {
   isConflictGroupType(groupType: Configurator.GroupType): boolean {
     return this.configuratorGroupsService.isConflictGroupType(groupType);
   }
+
+  createGroupId(groupId: string): string {
+    if (groupId) {
```

Comment:
If you really want to support undefined here, please have `createGroupId(groupId?: string)`